src/server/rest/v1/service/TenantService.ts

```diff
@@ -236,7 +236,7 @@ export default class TenantService {
     if (foundTenant) {
```

Comment:
I just saw that and to be honest we can remove this check.
There is no unicity on the tenant's name.

Reply:
Ok, removed